testutil/kubernetes_helper.go

```diff
@@ -54,6 +54,15 @@ func (h *KubernetesHelper) CheckIfNamespaceExists(namespace string) error {
 	return err
 }
 
+// GetSecret retrieves a Kubernetes Secret
+func (h *KubernetesHelper) GetSecret(namespace, name string) (*corev1.Secret, error) {
+	secret, err := h.clientset.CoreV1().Secrets(namespace).Get(name, metav1.GetOptions{})
```

Comment:
TIOLI: This is the same as `return h.clientset.CoreV1().Secrets(namespace).Get(name, metav1.GetOptions{})`.